ProcessMaker/modeler
# Add scrolling to controls panel

Fixes #186.

**How to test:**
 * Reduce the size of the window until it is not tall enough to fit all control elements.
 * Ensure a scrollbar appears to scroll any hidden control elements into view.

**Important changes:**
The `paper-container` ref has been removed, and the parent element height and width are now stored as data properties on the Modeler component so they can be easily applied to the controls component for sizing as well.
